test/inductor/test_torchinductor.py

```diff
@@ -238,6 +239,7 @@ def run_and_get_cpp_code(fn, *args, **kwargs):
     s = log_capture_string.getvalue()
     output_code_log.setLevel(prev_level)
     output_code_log.removeHandler(ch)
+    config.debug = old_debug_config
```

Comment:
safer to do "try ... finally" here to make sure `config.debug` is restored?

Reply:
I found that using the `patch` context manager instead of the decorator could fix this issue.
I've refined the code and added comments.